modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/config/ConfigFile.scala

```diff
@@ -54,6 +55,7 @@ final case class ConfigFile(
   output: Outputs,
   concurrency: Concurrency,
   assetsUpdatePeriod: Option[FiniteDuration],
+  remoteAdapters: Option[RemoteAdapterConfigs],
```

Comment:
How about make this non-optional?  The default value in `application.conf` could be:
```
"remoteAdapters": {
  "connectionTimeout": "10 seconds"
  "readTimeout": "45 seconds"
  "maxConnections": 10
  "configs: []
}
```

From point of view of the user it is still optional.  Meaning if the user does not care about remote adapters then they are allowed to not put anything extra in their config hocon.